src/coreclr/tools/aot/ILCompiler/ILCompilerRootCommand.cs

```diff
@@ -276,8 +276,8 @@ public ILCompilerRootCommand(string[] args) : base(".NET Native IL Compiler")
                     Console.ResetColor();
                     Console.ForegroundColor = ConsoleColor.Red;
 
+                    // Omit the stacktrace from the error (different from the default System.CommandLine exception handler)
                     Console.Error.WriteLine("Error: " + e.Message);
-                    Console.Error.WriteLine(e.ToString());
```

Comment:
Will this be printed by System.CommandLine instead?

Reply:
In Debug build, we get the stack, in Release build, it will print just the message.

Reply:
Message is not particularly useful - "Object reference not set to an instance of an object." is as good as printing "Internal compiler error". Could you please make it keep printing the stack? For crossgen2 as well.

Reply:
I noticed `Omit the stacktrace from the error (different from the default System.CommandLine exception handler)` in r2rdump and ilverify, I have changed them to show stacks as well (since this block is to catch unhandled exceptions).